src/core/components/ChatInput.tsx

```diff
@@ -16,6 +17,8 @@ const ChatInput = ({
   onCancel,
   setShouldAutoScroll,
 }: ChatInputProps) => {
+  const [showInputAdornmentMessage, setShowInputAdornmentMessage] =
```

Comment:
I imagine this state will eventually be managed in a hook that integrates with the utility to check the context limit. We could also display custom text based on how close the user is to going over the limit as well as other UI treatments, like disabling the send chat button.